scrapi/harvesters/vivo.py

```diff
@@ -0,0 +1,238 @@
+"""
+A VIVO harvester for the SHARE project
+
+This harvester makes a SPARQL query to the VIVO SPARQL endpoint,
+the information to acces to a VIVO endpoint must be provided in the local.py file.
+"""
+
+from __future__ import unicode_literals
+
+import json
+import logging
+
+from datetime import date, timedelta, datetime
+
+from six.moves import xrange
+from SPARQLWrapper import SPARQLWrapper, JSON
+
+from scrapi import settings
+from scrapi.settings.sparql_mapping import *
```

Comment:
Actually, having looked at the settings file now, why not just import that variable?
```python
from scrapi.settings.sparql_mapping import SPARQL_MAPPING
```